### PR TITLE
Add commit hash to front build

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This pull request includes the commit hash in our Next.js build to enable tracking of the app version our users are running in every SWR query. This feature will help us monitor refresh times, especially since we observed that most users don't refresh their screens after we refactored the conversation view.

In the future, we plan to implement a forced refresh mechanism based on the commit hash.

💡 **Side note:**
Next.js bundles environment variables prefixed with NEXT_PUBLIC_ at build time and makes them accessible to the UI.

Tested on front-edge.

image

### CODE REVIEW

**Review by lasryaric — APPROVED** [review]
LGTM

File: front/lib/swr.ts [hunk 1]
```diff
@@ -52,6 +52,8 @@ import type { GetProvidersResponseBody } from "@app/pages/api/w/[wId]/providers"
 import type { GetSubscriptionsResponseBody } from "@app/pages/api/w/[wId]/subscriptions";
 import type { GetWorkspaceAnalyticsResponse } from "@app/pages/api/w/[wId]/workspace-analytics";
 
+import { COMMIT_HASH } from "./commit-hash";
```
Comment (spolu) [inline front/lib/swr.ts:55]: This import should be global :pray:

Comment (spolu) [inline front/lib/swr.ts:55]: This import should be global :pray:

Comment (spolu) [reply front/lib/swr.ts:55]: Should be "@app/lib/commit-hash" ?

Comment (flvndvd) [reply front/lib/swr.ts:55]: Ah yeah 👍

**Pull request merged**